docs/HyperIndex/overview.md

```diff
@@ -7,34 +7,33 @@ slug: /overview
 
 HyperIndex is a modern, multi-chain data indexing framework for easily querying real-time and historical data from any EVM blockchain, as well as Fuel.
 
-Designed with a core focus on delivering superior performance and a seamless developer experience, HyperIndex optimizes the end-to-end user experience. 
+Designed with a core focus on delivering superior performance and a seamless developer experience, HyperIndex optimizes the end-to-end user experience.
 
 Build a real-time API for your blockchain application in minutes!
 
 ## Key Features
 
 :zap: **[Simplified Setup](/docs/HyperIndex/contract-import.md):** Start instantly with the quickstart, templates, and reference implementations.  
-:zap: **[Any EVM & Fuel](/docs/HyperIndex/supported-networks.md)**: Index smart contract data from any EVM-compatible network.  
```

Comment:
The `.md` is the only change in this file. It's now a folder with an index.md file